source/extensions/filters/http/common/pass_through_filter.h

```diff
@@ -18,10 +18,12 @@ class PassThroughDecoderFilter : public virtual StreamDecoderFilter {
   Http::FilterDataStatus decodeData(Buffer::Instance&, bool) override {
     return Http::FilterDataStatus::Continue;
   }
-
   Http::FilterTrailersStatus decodeTrailers(Http::HeaderMap&) override {
     return Http::FilterTrailersStatus::Continue;
   }
+  Http::FilterMetadataStatus decodeMetadata(Http::MetadataMap&) override {
```

Comment:
Revert all these changes in the filters now that you have a default?

Reply:
Fixed. Thanks!